x/payments/spec/03_messages.md

```diff
@@ -0,0 +1,145 @@
+# Messages
+
+In this section we describe the processing of the payment messages and the corresponding updates to the state. All created/modified state objects specified by each message are defined within the [state](./02_state.md) section.
+
+## MsgCreatePaymentTemplate
+
+PaymentTemplate can be created by any address using `MsgCreatePaymentTemplate`.
+
+| **Field**              | **Type**           | **Description**                                                                                               |
+|:-----------------------|:-----------------  |:--------------------------------------------------------------------------------------------------------------|
+| CreatorDid             | `did.DID`          | 
+| PaymentTemplate        | `PaymentTemplate`  | 
+
+```go
+type MsgCreatePaymentTemplate struct {
+	CreatorDid      did.Did         `json:"creator_did" yaml:"creator_did"`
+	PaymentTemplate PaymentTemplate `json:"payment_template" yaml:"payment_template"`
+}
+```
+
+This message creates and stores the `MsgCreatePaymentTemplate` object at appropriate indexes. 
+
+## MsgCreatePaymentContract 
+
+The owner of a bond can edit some of the payment's parameters using `MsgCreatePaymentContract`.
```

Comment:
This mentions bonds but we're not dealing with bonds here.

Reply:
changed to This will create a PaymentContract using `MsgCreatePaymentContract`.